wikibase/README.md

```diff
@@ -9,12 +9,12 @@ Automated build.
 
 ### Tags
 
-Image name                                                            | Parent image
------------------------------------------------------------------     | ------------------------
-`wikibase/wikibase` : `latest`, `1.34`, `latest-base`, `1.34-base`    | [mediawiki:1.34](https://hub.docker.com/_/mediawiki/)
-`wikibase/wikibase` : `1.31`, `lts`, `lts-base`, `1.31-base`          | [mediawiki:1.31](https://hub.docker.com/_/mediawiki/)
-`wikibase/wikibase` : `latest-bundle`, `1.34-bundle`                  | [wikibase:1.34](https://hub.docker.com/r/wikibase/wikibase/)
-`wikibase/wikibase` : `lts-bundle`, `1.31-bundle`                     | [wikibase:1.31](https://hub.docker.com/r/wikibase/wikibase/)
+Image name                                                                               | Parent image
+-------------------------------------------------------------------------------------    | ------------------------
+`wikibase/wikibase` : `latest`, `1.35`, `latest-base`, `1.35-base`, `lts`, `lts-base`    | [mediawiki:1.35](https://hub.docker.com/_/mediawiki/)
+`wikibase/wikibase` : `latest-bundle`, `1.35-bundle`, `lts-bundle`                       | [wikibase:1.35](https://hub.docker.com/r/wikibase/wikibase/)
+`wikibase/wikibase` : `1.34`, `1.34-base`                                                | [mediawiki:1.34](https://hub.docker.com/_/mediawiki/)
+`wikibase/wikibase` : `1.31`, `1.31-base`                                                | [mediawiki:1.31](https://hub.docker.com/_/mediawiki/)
```

Comment:
Looks like 1.31 and 1.34 are missing the bundle tags now (which are still being built)